extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -89,7 +91,20 @@ require([
 				adEngine.run(adConfigDesktop, win.ads.runtime.sp.slots, 'queue.sp');
 			});
 		}
+
 		sourcePoint.initDetection();
+
+		if (context.opts.recoveredAdsMessage) {
+			loader({
+				type: loader.AM_GROUPS,
+				resources: ['adengine2_ads_recovery_message_js']
+			}).done(function () {
+				require(['ext.wikia.adEngine.recovery.message'], function (recoveredAdMessage) {
+					recoveredAdMessage.init();
+				});
+			});
```

Comment:
If I'm not wrong, here will be made a new, separate request for `ext.wikia.adEngine.recovery.message` and probably it will be much easier to block this single request (with adblock rules) and break module.